apps/site/components/Search/SearchDropdownUsage.tsx

```diff
@@ -0,0 +1,145 @@
+import {
+  SearchDropdown,
+  SearchHistory,
+  SearchHistoryTerm,
+  SearchTop,
+  SearchTopTerm,
+  SearchAutoComplete,
+  SearchAutoCompleteTerm,
+  SearchProducts,
+  SearchProductItem,
+  SearchProductItemImage,
+  SearchProductItemContent,
+  SearchProvider,
+} from '@faststore/ui'
+
+import { useFormattedPrice } from 'site/components/utilities/usePriceFormatter'
+import { product } from 'site/mocks/product'
+
+export interface SearchDropdownUsageProps {
+  term: string
+  products?: [] | null
+}
+
+const SearchHistoryUsage = () => {
+  return (
+    <SearchHistory title="History">
+      <SearchHistoryTerm value="Headphone" linkProps={{ href: '/' }} />
+      <SearchHistoryTerm value="Audio & Video" linkProps={{ href: '/' }} />
+      <SearchHistoryTerm value="mh-7000" linkProps={{ href: '/' }} />
```

Comment:
Maybe using `#` as href will avoid the page redirect if user tries to click on, but it's literally **not** a blocker

Reply:
agreed!!!